Feature 2932 v12.0.0-beta5

jprestop opened the pull request:
## Expected Differences ##

- [x] Do these changes introduce new tools, command line arguments, or configuration file options? **[No]**</br>
If **yes**, please describe:</br>

- [x] Do these changes modify the structure of existing or add new output data types (e.g. statistic line types or NetCDF variables)? **[No]**</br>
If **yes**, please describe:</br>

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
None

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the beta5 release notes at this link: [MET Version 12.0.0-beta5 Release Notes (20240710)](https://met.readthedocs.io/en/feature_2932_v12.0.0-beta5/Users_Guide/release-notes.html#met-version-12-0-0-beta5-release-notes-20240710)
Pay particular attention to the items that are bolded.  I used my best judgement based on the priority tags listed on the issues, but would appreciate any feedback you have to give and requests for any changes.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the MET test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Will this PR result in changes to existing METplus Use Cases? **[No]**</br>
If **yes**, create a new **Update Truth** [METplus issue](https://github.com/dtcenter/METplus/issues/new/choose) to describe them.

- [x] Do these changes introduce new SonarQube findings? **[No]**</br>
If **yes**, please describe:

- [x] Please complete this pull request review by **[20240710 AM]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the version that will include these changes
Select: **Coordinated METplus-X.Y Support** project for bugfix releases or **MET-X.Y.Z Development** project for official releases
- [x] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

jprestop committed:
Per #2932, updating version and release notes

jprestop committed:
Per #2932, updating date on release notes

jprestop committed:
Per #2932, fixed formatting and links

JohnHalleyGotway committed:
Update release-notes.rst

JohnHalleyGotway commented:
Alright Julie, thanks for putting together these release notes. I made some significant changes, listed below:
- Fixed one minor typo `#3717` to `#2717`.
- Moved METbaseimage issue to the top of the list to keep them organized by repo. I'm torn on whether it belong here since we do have release notes in this [README](https://github.com/dtcenter/METbaseimage/blob/main/README.md) file. But I'll just leave it since you put it here.
- Reworded titles of all bugfix issues/release notes to begin with `Fix <TOOL NAME> ...`.
- Reworded other issue titles for consistency/clarity.
- Reordered the notes to put the issues in numeric order within each section.
- Unbolded several enhancements to minimize the number of bolds. If "everything" is emphasized, then nothing is. I generally highlight new command line options, config file options, new statistics output, and major changes to functionality. But this is very subjective.
- Added back in back-ticks around config/command line options to format them nicely in the release notes. They don't survive copy/paste generally.

Here's the [commit](https://github.com/dtcenter/MET/commit/1854eaffa9e6e2de2d3196e80ff834877ae4f7f4) with those changes.

JohnHalleyGotway reviewed (dismissed):
Julie, with the changes I added, I'm happy with these release notes and approve.

But I'll put the ball back in your court. Please read through them again and merge once you're also happy.

Thanks for your work on this.

JohnHalleyGotway committed:
Update release-notes.rst

Removing inline backticks since they do not format the way I expected, especially when put inside bolded release notes.

JohnHalleyGotway commented:
FYI, I just removed the inline backticks because do not format the way I expected, especially when used inside release notes that are bolded.

JohnHalleyGotway reviewed (approved):
I approve of these changes.
